usr/src/cmd/fs.d/nfs/mountd/mountd.c

```diff
@@ -362,6 +363,20 @@ convert_int(int *val, char *str)
 	return (0);
 }
 
+static void
+create_svcs(const rpcprog_t prognum, const rpcvers_t versnum, int port) {
```

Comment:
Probably need to double check pbchk. Should catch that this should be on a new line.